packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1667,6 +1729,11 @@ class StackViewport extends Viewport implements IStackViewport {
           this._cpuFallbackEnabledElement.viewport.colormap
         );
 
+        const { windowCenter, windowWidth } = viewport.voi;
+        this.voiRange = {
```

Comment:
There is no need to create a secondary object to hold this return as toLowHighRange returns a new object every time.